bitcoin/src/blockdata/witness.rs

```diff
@@ -139,22 +139,19 @@ crate::internal_macros::define_extension_trait! {
             self.push(signature.serialize())
         }
 
-        /// Get Tapscript following BIP341 rules regarding accounting for an annex.
+        /// Returns the leaf script with its version but without the merkle proof.
```

Comment:
nit: `Returns the leaf script with it's version and no merkle proof.` is easier to read imo

Reply:
"it's" is short for "it is" but this is not "it is"

Reply:
> "it's" is short for "it is" but this is not "it is"

hah good point.

Reply:
I meant the possessive noun `its` not the contraction, my mistake.